src/plots/barPlot.ts

```diff
@@ -113,6 +120,56 @@ export class Bar<X, Y> extends XYPlot<X, Y> {
     return this;
   }
 
+  public barEnd(): Plots.TransformableAccessorScaleBinding<X, number>;
```

Comment:
There is definitely overlap insofar as you could make EVERY bar chart with a rectangle plot. But, implementing this in bar plot gives us stacked bar histograms for free.